imageroot/actions/configure-module/validate-input.json

```diff
@@ -7,16 +7,25 @@
         {
             "synapse_domain_name": "matrix.example.com",
             "element_domain_name": "chat.example.com",
+            "cinny_domain_name": "cinny.example.com",
             "lets_encrypt": true,
-            "dex_ldap_domain": "users.example.com"
+            "dex_ldap_domain": "users.example.com",
+            "mail_from": "noreply@example.com",
+            "enable_element": true,
+            "enable_cinny": false,
```

Comment:
This should not be needed, we can reuse `cinny_domain_name` and `element_domain_name` as flags: if they are not empty, just start the services